mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Always aim to use the latest available EAPI, 8 should work with this ebuild.

Reply:
Fixed. Thanks!